File: Part4/Go/Function_IgnoredReturn.go
package main

import (
	"fmt"
	"strconv"
)

func main() {
	var userInput string
	var enteredInteger int

	fmt.Println( "Type 1 and press enter." )
	fmt.Scanln( &userInput )

	// Function called but return value ignored
	strconv.Atoi( userInput )

	fmt.Printf( "The user entered the integer %d\n", enteredInteger )
}

File: Part4/C++/Function_IgnoredReturn.cpp
#include <stdio.h>
#include <stdlib.h>

int main()
{
    char userInput[ 100 ];
    int enteredInteger;
    
    printf( "Type 1 and press enter.\n" );
    fgets( userInput, sizeof( userInput ), stdin );
    
    // Function called but return value ignored
    atoi( userInput );
    
    // enteredInteger was never assigned a value
    printf( "The user entered the integer %d\n", enteredInteger );
    
    return 0;
}
